repo/packages/H/hdfs/0/command.json

```diff
@@ -0,0 +1,5 @@
+{
+  "pip": [
+    "git+https://github.com/mesosphere/dcos-hdfs.git#dcos-hdfs=0.1.0"
```

Comment:
> what is the 0.1.0 point to though?
Yes. Pip URI can be confusing. Don't quote me on it but the `#dcos-hdfs=0.1.0` is use by pip during dependency resolution. It is up to you to make sure that the default branch of the git repo contains the version for 0.1.0. One way around this to have tags and then you can do something like `...git@0.1.0#dcos-hdfs=0.1.0`